### ISSUE TITLE
swipe for mobile viewport

### ISSUE THREAD

**Issue by prashant114606-zz** [viljamis/ResponsiveSlides.js#384]
Hi I was just looking for why it doesn't support swipe functionality for mobile view but it seems it is not defined in library I think giving to to swipe make more useable and will be helpful if you fix this asap

**Comment by DVeris** [comment]
+1

**Comment by pixelstuff** [comment]
Yes, this would be helpful to have. Assuming it doesn't add to much size to the code. Could the [jQuery TouchSwipe plugin](https://github.com/mattbryson/TouchSwipe-Jquery-Plugin) be co-opted to send its swipe detection back to ResponsiveSlides?

I guess, alternatively, for someone reading this later there is swiperjs.com that has all the bells and whistles.
